src/raft.rs

```diff
@@ -593,13 +584,14 @@ impl<T: Storage> Raft<T> {
     pub fn maybe_commit(&mut self) -> bool {
         let mut mis_arr = [0; 5];
         let mut mis_vec;
-        let mis = if self.prs().voters().len() <= 5 {
-            &mut mis_arr[..self.prs().voters().len()]
+        let voters = self.prs().voters().count();
```

Comment:
why not providing a num_voter function so we don't need to calculate the voters from the iterator.